docs/pbc/all/search/202212.0/third-party-integrations/algolia.md

```diff
@@ -4,6 +4,8 @@ description: Algolia empowers Builders with Search and Recommendation services t
 template: howto-guide-template
 ---
 
+ADD HERE THE ALGOLIA-HERO PICTURE
```

Comment:
@lenadoc Picture to replace here from the Algolia folder

Reply:
@jeremyfourna I've moved pictures to S3 and removed them from our portal. Links added.